docs/guides/src/main/server/reverseproxy.adoc

```diff
@@ -43,6 +43,15 @@ Take extra precautions to ensure that the X-Forwarded-For header is set by your
 If this header is incorrectly configured, rogue clients can set this header and trick Keycloak into thinking the client is connected from a different IP address than the actual address.
 This precaution can more be critical if you do any deny or allow listing of IP addresses.
 
+=== Exposing the administration console
+
+By default, the administration console URLs are created solely based on the requests to resolve the proper scheme, host name, and port. For instance,
+if you are using the `edge` proxy mode and your proxy is misconfigured, backend requests from your TLS termination proxy are going to use plain HTTP and potentially cause the administration
```

Comment:
understanding question: 
`edge` implies for me that TLS termination is done at the edge, as the name actually suggests. so is it really a misconfiguration when the proxy forwards internal requests  using `HTTP`? Or do you mean that it's a misconfiguration that it forwards using `HTTP` _but_ no `X-Forwarded-Proto: https` or `X-Forwarded-Scheme: https` is used?

Reply:
It is a misconfiguration if you don't properly set the headers. It should be clear if you read the rest of the section. Or do you want to suggest another way of saying the same?